pkg/target/target.go

```diff
@@ -130,7 +128,7 @@ func (h *K8sValidationTarget) handleReview(obj interface{}) (bool, *gkReview, er
 		return false, nil, nil
 	}
 
-	if err := setObjectOnDelete(review); err != nil {
+	if err := util.SetObjectOnDelete(&review.AdmissionRequest); err != nil {
```

Comment:
This will be executed twice for admission requests going through webhook. Once in policy.go and once in target handler. Best way to fix this is probably to revert the change and not have it under target handler or use oldObject when object is nil in `skipExcludedNamespace`.

Reply:
then I'll go for the second option, i.e. use oldObject when object is nil (and we are having to do with a delete op) in skipExcluded namespace, as I imagine there were some reasons to move it to target in the first place